lib/api.js

```diff
@@ -66,16 +67,21 @@ function _authenticate(options) {
   return {'basic': 'Basic ' + base64Encoded};
 }
 
+function _getDefaultUserAgent() {
+  return [
+    'OmiseNode/', pkgjson.version, ' ',
+    'Node/', process.version,
+  ].join('');
+}
+
 function _prepareOptionsFor(method, options) {
-  var content = _buildContentHeaders(options)();
-  var headers = {
+  const content   = _buildContentHeaders(options)();
+  const userAgent = options['userAgent'] || _getDefaultUserAgent();
```

Comment:
@ajzkk Great work here. Regarding user agent, as per the chagne, we are using only the value set in `userAgent` as user agent. Ideally, we would want to append the value of `userAgent` to the default user agent. We should have our user agent value as `OmiseNode/0.9.0 Node/16.19.0 ShopifyPaymentApp/v1.9.1` instead of just `ShopifyPaymentApp/v1.9.1`. If no `userAgent` is set then user agent value will be `OmiseNode/0.9.0 Node/16.19.0`.

Reply:
Fixed